pkg/api/v1alpha1/cluster.go

```diff
@@ -494,6 +495,16 @@ func validateControlPlaneCertSANs(cfg *Cluster) error {
 	return nil
 }
 
+func validateControlPlaneAPIServerExtraArgs(clusterConfig *Cluster) error {
+	allowedFlags := map[string]string{"service-account-issuer": "", "service-account-jwks-uri": ""}
```

Comment:
Let's add a feature flag to disable this validation, should be a simple change as we discussed in doc right?

Reply:
Do you mean a feature flag or an addition to skip validation?

Reply:
Feature flag

Reply:
Feature flags should be short lived and have a rough criteria defining when they can be removed.

Looking at the change set, a feature flag isn't justified here. Instead, I think you want to be able to disable the validation, which is a permanent feature.

Reply:
I reviewed the design doc. Sorry I didn't catch this when reviewing: I don't think we meet the criteria for a feature falg. This is a feature that we may deprecate in the future.

Reply:
Removed feature flag and added api server extra args validation to the `skip-validations` option